scrapper/trino/query_table_metrics.go

```diff
@@ -0,0 +1,66 @@
+package trino
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"time"
+
+	"github.com/getsynq/dwhsupport/scrapper"
+)
+
+type trinoStatsRow struct {
+	ColumnName         sql.NullString  `db:"column_name"`
+	RowCount           sql.NullFloat64 `db:"row_count"`
+	DataSize           sql.NullFloat64 `db:"data_size"`
+	DistinctValueCount sql.NullFloat64 `db:"distinct_values_count"`
+	NullsFraction      sql.NullFloat64 `db:"nulls_fraction"`
+	LowValue           sql.NullString  `db:"low_value"`
+	HighValue          sql.NullString  `db:"high_value"`
+}
+
+func (e *TrinoScrapper) QueryTableMetrics(ctx context.Context, lastMetricsFetchTime time.Time) ([]*scrapper.TableMetricsRow, error) {
+	tables, err := e.QueryTables(ctx)
+	if err != nil {
+		return nil, err
+	}
+	db := e.executor.GetDb()
+	var out []*scrapper.TableMetricsRow
+
+	for _, t := range tables {
+		fqTable := fmt.Sprintf("%s.%s.%s", t.Database, t.Schema, t.Table)
+		query := fmt.Sprintf("SHOW STATS FOR %s", fqTable)
+		rows, err := db.QueryxContext(ctx, query)
+		if err != nil {
+			return nil, err
+		}
+		// Close rows as soon as possible
+		func() {
+			defer rows.Close()
+			for rows.Next() {
+				var stat trinoStatsRow
+				if err := rows.StructScan(&stat); err != nil {
+					return
+				}
+				if !stat.ColumnName.Valid { // NULL column_name row
```

Comment:
waiting to NULL named column to get `row_count`

![CleanShot 2025-05-09 at 13 23 48@2x](https://github.com/user-attachments/assets/db244bf9-0c11-452d-b46c-f5493413f4b0)